pyorbital/tlefile.py

```diff
@@ -106,6 +106,44 @@ def read(platform, tle_file=None, line1=None, line2=None):
     return Tle(platform, tle_file=tle_file, line1=line1, line2=line2)
 
 
+def get_all_platforms(tle_file=None):
+    """Read all *platform* name from tle file
+    tle_file - list path files
+    """
+    platforms = list()
+    fid = io.open(tle_file, 'rb')
+    for line in fid:
+        line = line.decode('utf-8')
+        line = line.strip()
+        if not line.startswith('1') and not line.startswith('2') and len(line) > 0:
+            platforms.append(line)
+            l_1 = next(fid).decode('utf-8')
+            l_2 = next(fid).decode('utf-8')
+    fid.close()
+
+    return platforms
+
+
+def get_platform(tle_file=None):
+    """Read *platform* name from tle file
+    tle_file - path to tle file
+    """
+    try:
+        fid = io.open(tle_file, 'rb')
```

Comment:
What about getting rid of the `try/except/finally` and instead using `with io.open(tle_file, 'rb') as fid:`?

Reply:
Of course, it is really worked